xmovie/core.py

```diff
@@ -372,6 +438,8 @@ def save(
         progress=False,
```

Comment:
If you agree with the above suggestion to remove dask progress bar, this should be mentioned here.

Reply:
Done!